cli/chaosli/cmd/create.go

```diff
@@ -0,0 +1,485 @@
+
+	spec.Hosts = getHosts()
+	spec.Services = getServices()
+
+	spec.Flow, _ = selectInput(
+		"Choose a flow direction",
+		[]string{v1beta1.FlowEgress, v1beta1.FlowIngress},
+		fmt.Sprintf("%s will affect traffic leaving the target. %s will not really affect traffic entering the target, but actually will affect replies to the inbound traffic.",
+			v1beta1.FlowEgress, v1beta1.FlowIngress),
+	)
+
+	if confirmOption("Would you like to drop packets?", "Packets will be dropped before leaving the target") {
+		spec.Drop, _ = strconv.Atoi(getInput("What % of packets should we affect?", "1-100", true))
+	}
+
+	if confirmOption("Would you like to duplicate packets?", "Packets will be duplicated immediately before leaving the target") {
+		spec.Duplicate, _ = strconv.Atoi(getInput("What % of packets should we affect?", "1-100", true))
+	}
+
+	if confirmOption("Would you like to corrupt packets?", "Packets will be corrupted before leaving the target") {
+		spec.Corrupt, _ = strconv.Atoi(getInput("What % of packets should we affect?", "1-100", true))
+	}
+
+	if confirmOption("Would you like to delay packets?", "Packets will be delayed before leaving the target") {
+		delay, _ := strconv.ParseUint(getInput("How much to delay (in ms)?", "This will be the median amount of delay to apply", true), 10, 0)
+		spec.Delay = uint(delay)
+
+		delayJitter, _ := strconv.ParseUint(getInput("What jitter on that delay (in ms)?", "This will be normally distributed around the delay you specified earlier. This will cause packets to re-order!", false), 10, 0)
+		spec.DelayJitter = uint(delayJitter)
+	}
+
+	if confirmOption("Would you like to limit bandwidth?", "bandwidthlimit") {
+		spec.BandwidthLimit, _ = strconv.Atoi(getInput("What bandwidth limit should we set (in bytes per second)?", ">0", true))
+	}
+
+	return spec
+}
+
+func getContainers() []string {
+	if !confirmOption("Would you like to target a specific container[s]?", "The default is to target all containers in the target pod.") {
+		return nil
+	}
+
+	containers := getSliceInput("Please enter a comma-delimited list of container name[s] to target.", "Please specify their names, not their IDs!", false)
+
+	return containers
+}
+
+func getCount() *intstr.IntOrString {
+	result := getInput(
+		"How many targets would you like to disrupt? This can be an integer, or a percentage.",
+		"Please specify an integer >0 or a percentage from 1% - 100%. If specifying a percentage, you must suffix with the % character, or we will think its an integer!",
+		true,
+	)
+	// TODO somehow grab the other intrstr validate here
+
+	wrappedResult := intstr.FromString(result)
+
+	return &wrappedResult
+}
+
+func getSelectors() labels.Set {
+	selectors := getSliceInput(
+		"Add a label selector[s] for targeting.",
+		"Please specify this in the form of `key=value`, e.g., `app=hello-node`. One label selector per new-line. If you specify multiple, we will only target the union.",
```

Comment:
I think an example of how multiple selectors should be added to the cli would be really helpful. The text is helpful in itself, but for visual learners, just seeing the example will be easier to digest. Will not lie, as simple as your explanation was, still found myself reading it twice to understand.